.gitignore

```diff
@@ -11,12 +11,14 @@
 
 # misc
 .DS_Store
+.env
 .env.local
 .env.development.local
 .env.test.local
 .env.production.local
 
 npm-debug.log*
+yarn.lock
```

Comment:
@callumflack So I think the only thing remaining then is to remove the actual yarn.lock from the PR, probably via `git rm yarn.lock`, commit/push, then put it right back locally and git should ignore it from here on.

Reply:
Again, d'oh. Pushed ;)